ext/drmgr/drmgr.c

```diff
@@ -652,6 +652,7 @@ drmgr_bb_event(void *drcontext, void *tag, instrlist_t *bb,
             e = &iter_insert.cbs.bb[i];
             if (!e->pri.valid)
                 continue;
+            instrlist_set_auto_predicate(bb, instr_get_predicate(inst));
```

Comment:
Please add a subsection to drmgr.dox explaining this, just like there's one on IT blocks